data/campaigns/Northern_Rebirth/scenarios/12a_Get_the_Gold.cfg

```diff
@@ -574,13 +582,13 @@
             [then]
                 [message]
                     id=Tallin
-                    message= _ "Thanks to Hidel and the elves, we have recovered our gold. Now let’s run down Rakshas and settle up with him once and for all."
+                    message= _ "Thanks to Hidel and the elves, we have recovered our gold. Now let’s run down Rakshas end this once and for all."
                 [/message]
             [/then]
             [else]
                 [message]
                     id=Tallin
-                    message= _ "Hidel’s death was a grievous loss; our thirst for vengeance must do what the stolen gold cannot. Now let’s run down Rakshas and settle up with him once and for all."
+                    message= _ "Hidel’s death was a grievous loss; our thirst for vengeance must do what the stolen gold cannot. Now let’s run down Rakshas and settle this once and for all."
```

Comment:
Agreed with removing the redundant wording here and line 585, but this is an either-or situation - perhaps make it consistent to have both say 'settle' or both say 'end'?

Reply:
It makes certain, if you play the story through a different way, it's not all the same. (Joke)